plugins/gatsby-plugin-elasticlunr-search-fiori/gatsby-node.js

```diff
@@ -0,0 +1,194 @@
+"use strict"
+const webVersion = require('.../src/pages/versions/web-version.json')
+const iosVersion = require('.../src/pages/versions/ios-version.json')
+const androidVersion = require('.../src/pages/versions/android-version.json')
+const cuxVersion = require('.../src/pages/versions/android-version.json')
+
+
+function _objectSpread(target) {
+    for (var i = 1; i < arguments.length; i++) {
+        var source = arguments[i] != null ? arguments[i] : {}
+        var ownKeys = Object.keys(source)
+        if (typeof Object.getOwnPropertySymbols === `function`) {
```

Comment:
This won't be true always?

Reply:
Not sure. Prefer to leave this as is since it was copied form @gatsby-contrib/gatsby-plugin-elasticlunr-search